rest_api/openshift_v1.adoc

```diff
@@ -10284,6 +10284,7 @@ ObjectReference contains enough information to let you inspect or modify the ref
 |noCache|if true, indicates that the Docker build must be executed with the --no-cache=true flag|false|boolean|false
 |env|additional environment variables you want to pass into a builder container|false|<<v1.EnvVar>> array|
 |forcePull|forces the source build to pull the image if true|false|boolean|false
+|dockerfilePath|path of the Dockerfile to use for building the Docker image, relative to the contextDir, if set|false|string|
```

Comment:
Path. Could be somedir/somedockerfile.
Le 15 déc. 2015 15:46, "Vyacheslav Semushin" <notifications@github.com> a
écrit :

> In rest_api/openshift_v1.adoc
> <https://github.com/openshift/openshift-docs/pull/1345#discussion_r47643475>
> :
>
> > @@ -10284,6 +10284,7 @@ ObjectReference contains enough information to let you inspect or modify the ref
> >  |noCache|if true, indicates that the Docker build must be executed with the --no-cache=true flag|false|boolean|false
> >  |env|additional environment variables you want to pass into a builder container|false|<<v1.EnvVar>> array|
> >  |forcePull|forces the source build to pull the image if true|false|boolean|false
> > +|dockerfilePath|path of the Dockerfile to use for building the Docker image, relative to the contextDir, if set|false|string|
>
> Is it path or file name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/openshift-docs/pull/1345/files#r47643475>.
>